src/WasmExecutor.cpp

```diff
@@ -2327,11 +2311,11 @@ WasmModuleContents::WasmModuleContents(
         << wabt::FormatErrorsToString(errors, wabt::Location::Type::Binary) << "\n"
         << "  log: " << to_string(log_stream) << "\n";
 
-    if (WASM_DEBUG_LEVEL >= 2) {
+    if (debug_is_active(2)) {
         wabt::MemoryStream dis_stream;
         module_desc.istream.Disassemble(&dis_stream);
-        wdebug(WASM_DEBUG_LEVEL) << "Disassembly:\n"
-                                 << to_string(dis_stream) << "\n";
+        debug(2) << "Disassembly:\n"
+                 << to_string(dis_stream) << "\n";
     }
```

Comment:
I'd ultimately like to remove uses of the `debug_is_active` helper as it doesn't play nice with line-based filters. Here's an alternative:
```cpp
    debug(2) << "Disassembly:\n" << [&] {
        wabt::MemoryStream dis_stream;
        module_desc.istream.Disassemble(&dis_stream);
        return to_string(dis_stream);
    }() << "\n";
```